_posts/2019-10-3-WaiS.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: WaiS
+---
+
+![WaiS](https://www.google.com/url?sa=i&source=images&cd=&ved=2ahUKEwiK18aigYDlAhVLH48KHSWTCo0QjRx6BAgBEAQ&url=http%3A%2F%2Fwww.drawastickman.com%2F&psig=AOvVaw1OBzLkx1L1LKyFczgD_izM&ust=1570189343613660)
```

Comment:
Image link does not work (does not directly link to image).

Reply:
Please update with a direct link that ends in an extension (`.png`, `.jpg`, `.gif`, etc)